src/rabbit_amqqueue_process.erl

```diff
@@ -142,6 +142,8 @@ init_it(Recover, From, State = #q{q = #amqqueue{exclusive_owner = Owner}}) ->
                  {_, Terms} = recovery_status(Recover),
                  BQS = bq_init(BQ, Q, Terms),
                  %% Rely on terminate to delete the queue.
+                 log_auto_delete("Exclusive query connection closed", 
```

Comment:
You probably mean "queue", not "query". Also, this conflates exclusive and auto-delete queues. That's confusing, please use separate functions with clearer wording (as discussed on Slack).